src/App.scss

```diff
@@ -36,25 +36,21 @@ body {
     flex-direction: column;
 }
 
-.App__header-keyboardMenuIcon {
-    align-content: center;
-    align-self: center;
-    border-radius: 50%;
-    background-color: #4C9990;
-    font-size: 1.6rem;
-    text-align: center;
-    vertical-align: middle;
-    margin-bottom: 0.5rem;
-    height: 2.6rem;
-    width: 2.6rem;
-
-    &:hover {
-        background-color: #F1AE5B;
-    }
+.App__header-menu {
+    // Uncomment background-color and border radius later when we change all the setting
+    // menu items as buttons
+    // background-color: #CCE6E2;
+    // border-radius: 3px;
+    display: grid;
+    grid-template-columns: min-content min-content;
+    justify-content: center;
+    column-gap: 0.5rem;
+    padding: 0.5rem 0;
+}
 
-    &:focus {
-        box-shadow: 0 0 0 $c2lc-focus-indicator-width black;
-        outline: none;
+.IconButton.App__header-soundOptions {
+    svg path {
```

Comment:
You could collapse the selector as there's nothing else being updated.

Reply:
I took out the rule specified to the keyboard modal icon button to be styled by general IconButton styles specified in App__header-menu.